### PR TITLE
Add support for actions

### PR DESCRIPTION
**Pull request by fbiville**

### CODE REVIEW

File: src/main/resources/spec.v1.0.json [hunk 1]
```diff
@@ -659,6 +659,13 @@
         "query": {
           "type": "string",
           "minLength": 1
+        },
+        "execution_mode": {
+          "type": "string",
+          "enum": [
+            "transaction",
+            "autocommit"
```
Comment (ali-ince) [inline src/main/resources/spec.v1.0.json:667]: is this solely for supporting `CALL IN TRANSACTIONS`?

File: src/test/java/org/neo4j/importer/v1/e2e/BeamIT.java [hunk 2]
```diff
@@ -86,6 +99,10 @@ public class BeamIT {
     public void prepare() {
         neo4jDriver = GraphDatabase.driver(NEO4J.getBoltUrl(), AuthTokens.basic("neo4j", NEO4J.getAdminPassword()));
         neo4jDriver.verifyConnectivity();
+        try (Session session = neo4jDriver.session(
+                SessionConfig.builder().withDatabase("system").build())) {
+            session.run("CREATE OR REPLACE DATABASE neo4j").consume();
```
Comment (ali-ince) [inline src/test/java/org/neo4j/importer/v1/e2e/BeamIT.java:104]: I have ran into some issues if we don't specify `WAIT` at the end of our DDL statements, suggest we add it here too.

**Review by ali-ince — APPROVED** [review]
looks good to me. left a couple of small comments.

File: src/main/resources/spec.v1.0.json (hunk 1 above)
Comment (fbiville) [reply src/main/resources/spec.v1.0.json:667]: yes, I forgot to put it in the schema, but it was already in `CypherAction`

Comment (fbiville) [reply src/main/resources/spec.v1.0.json:667]: the default value is `transaction` so people don't have to worry about it unless they're trying CIT

File: src/test/java/org/neo4j/importer/v1/e2e/BeamIT.java (hunk 2 above)
Comment (fbiville) [reply src/test/java/org/neo4j/importer/v1/e2e/BeamIT.java]: fixed in 25f270385b3adda0ea8c76281435d40fbf8cd4ea

**Review by ali-ince — APPROVED** [review]

**Pull request merged**